### PR TITLE
Update yml and makefile

### PR DESCRIPTION
**Pull request by shiara02**
### OVERVIEW

-Changed the ports
-Edited the makefile commands

### CODE REVIEW

**Review by hipe-francis — DISMISSED** [review]
LGTM 🚀

File: Makefile [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
-	docker-compose exec -it backend php artisan module:seed
+    if ! [ -f ./src/backend/.env ];then cp ./src/backend/.env.example ./src/backend/.env;fi
+    docker compose exec -it backend php artisan key:generate
+    docker compose exec -it backend php artisan optimize:clear
+    docker compose exec -it backend php artisan migrate --seed
 
 optimize:
-	docker-compose exec -it backend php artisan optimize:clear
+    docker compose exec -it backend php artisan optimize:clear
+
```
Comment (hipe01mark) [inline Makefile]: Trail spacing!

File: docker-compose.yml [hunk 2]
```diff
@@ -44,38 +44,39 @@ services:
...
-      - 13082:80
+      - 43081:80
     environment:
       PMA_HOST: database
     depends_on:
       - database
     networks:
-      - app
+      - microblog
+
```
Comment (hipe01mark) [inline docker-compose.yml]: Trail spacing!

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by hipe-francis — COMMENTED** [review]
LGTM 🚀

**Review by hipe-francis — APPROVED** [review]

**Pull request merged**